unlock-app/src/components/interface/layouts/index/DashboardHeader.tsx

```diff
@@ -38,8 +26,59 @@ export default function DashboardHeader({
   const { openConnectModal } = useConnectModal()
   const pathname = usePathname()
 
-  const menuProps =
-    showMenu && pathname !== '/' ? MENU : { ...MENU, menuSections: [] }
+  // Determine logo config based on pathname
+  const getLogo = () => {
+    if (pathname?.includes('subscription')) {
+      return {
+        url: '/subscription',
+        src: '/images/svg/logo-unlock-subscriptions.svg',
+      }
+    }
+    if (pathname?.includes('event')) {
+      return {
+        url: '/event',
+        src: '/images/svg/logo-unlock-events.svg',
+      }
+    }
+    if (pathname?.includes('certification')) {
+      return {
+        url: '/certification',
+        src: '/images/svg/logo-unlock-certificate.svg',
+      }
+    }
+    return {
+      url: '/',
+      src: '/images/svg/unlock-logo.svg',
+    }
+  }
+  console.log('pathname', pathname)
+
+  // Determine menu sections based on pathname
+  const getMenuSections = () => {
+    if (pathname?.includes('event')) {
+      return [{ title: 'Events', url: '/my-events' }]
```

Comment:
I actually think we should show the same menus everywhere...

Reply:
alright, got it!